streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -482,7 +482,7 @@ public void shouldReturnFalseForRestoreActiveTasksIfTaskPaused() throws Exceptio
         verifyRemovedTasks();
         verifyPausedTasks(task);
 
-        assertFalse(stateUpdater.restoresActiveTasks());
+        assertTrue(stateUpdater.restoresActiveTasks());
```

Comment:
This is related to the change making sure with and without state updater has consistent behavior.